Homework.html

```diff
@@ -0,0 +1,67 @@
+    name : "Women's Dress Sleeveless",
+    description : " Sleeveless Cocktail Dresses Knee Length Evening Dresses Elegant Lace Dress V-Neck Asymmetrical Bridesmaid Dress",
+    Price: "37€" ,
+     shippingCost :{
+         cost : freeShipping
+     }
+},
+{
+    imageUrl : "https://images-na.ssl-images-amazon.com/images/I/81Yq1G6O-CL._AC_UY445_.jpg" ,
+    name : "Women's Maxi Floral dress",
+    description : " Multicolored casual Floral dress with line",
+    Price: "25€"  ,
+     shippingCost :{
+         cost : freeShipping
+     }
+},
+{
+    imageUrl : "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRF1avcP5Nli-Mu5mYvk1HvIhqu8QpFFumFonCHNNBwmuP9_1f4Qox_dJLTiM9ep58ZFTXEcHE&usqp=CAc",
+    name : "Women's Summer dress",
+    description : " Long Casual Elegant Dresses Party Dress Beach Dress",
+    Price: "35€", 
+     shippingCost :{
+         cost : freeShipping
+     }
+},
+{
+    imageUrl : "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRMancKgpx0vgr9eGIYERMONz6vWT-F5XuMgWmBnrjbOD04KCmQ0KYi-lEVsTrPe6hb264ZLrhU&usqp=CAc",
+    name : "Women's Nightie",
+    description : " Short,pleasantly loose cut which is 100% cotton",
+    Price: "15€", 
+     shippingCost :{
+         cost : freeShipping
+     }
+}
+
+] ;
+
+function addImage () {
+    let newImage = document.createElement("img");
+    newImage.src = dressList[1].imageUrl;
+    firstItemCmp.appendChild(newImage);
+};
+
+dressListCmp.innerHTML = `<li>${dressList[3].Price}</li>`;
```

Comment:
if you wanted to include an image you could do 
```
dressListCmp.innerHTML = `
    <li>
        <p>Price:${dressList[3].Price}</p>
        <img src="${dressList[3].imageUrl}" />
    </li>
`;
```

resulting in 
<img width="533" alt="Screenshot 2020-04-26 at 12 03 11" src="https://user-images.githubusercontent.com/13404719/80304419-f48e0580-87b5-11ea-8f8b-b8256b5dedf3.png">